cid/helpers/athena.py

```diff
@@ -595,7 +595,7 @@ def find_tables_with_columns_in_information_schema(self, columns):
         ''')
 
     def find_tables_with_columns(self, columns: list, database_name: str=None, catalog_name: str=None, max_items: int=10000):
-        """ This function searches a table with a given set of columns
+        """ This function return interator of tables with a given set of columns.
```

Comment:
I think there is typo here and overall description kind of doesn't make sense.

Reply:
1. function returns interator
2. iterrator that iterates tables 
3. but not all. only those that have a given set of columns 
 
 any suggestion how to phase it better?

Reply:
gen ai suggestion: 

Returns an iterator that yields only tables containing all specified columns.

Reply:
```suggestion
        """ Returns an iterator that yields only tables containing all specified columns.
```